Source/WebCore/platform/graphics/coretext/FontCoreText.cpp

```diff
@@ -394,6 +394,8 @@ bool Font::supportsAllPetiteCaps() const
 }
 #endif
 
+
+// todo: should we adjust font for font-size-adjust in derivative font case?
```

Comment:
We use FIXME, not todo, in this project, and we capitalize the sentence, "FIXME: Should [...]".

Reply:
Usually these kind of things are accompanied by a bug link.